authorization/index.md

```diff
@@ -498,6 +537,10 @@ redirect_uri=https%3A%2F%2Fapp%2Fafter-auth
   "intent": "client-ui-name",
   "patient":  "123",
   "encounter": "456"
+  "host": "https://hospital.org/EHR_authsvr/endpoint", 
+  "client_id":  "123abcf8",
+  "username": "QBOEKARUAOE",
```

Comment:
@bakerdb I think we've got wires crossed on this point -- the response to the authorization request is an `application/json` body, not a JWT, and these aren't parameters we've documented elsewhere. Especially "username" which clearly belongs in the `id_token` (which is a JWT, and is one of the properties of the response object).